[22.08.2024, 10:11:15] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:16] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:16] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:16] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:17] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:17] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:17] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:18] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:18] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:18] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:19] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:19] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:19] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:20] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:11:21] RequestError: Error: connect EHOSTUNREACH 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:28:48] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:30:17] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:30:32] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:32:59] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:33:06] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:33:31] Error: ETELEGRAM: 400 Bad Request: message is not modified: specified new message content and reply markup are exactly the same as a current content and reply markup of the message
    at C:\web\chat-bot-telegram\node_modules\node-telegram-bot-api\src\telegram.js:316:15
    at tryCatcher (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (C:\web\chat-bot-telegram\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)
[22.08.2024, 10:49:14] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:16] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:18] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:21] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:23] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:26] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[22.08.2024, 10:49:28] RequestError: Error: connect ECONNREFUSED 169.254.0.51:3274
    at new RequestError (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\errors.js:14:15)
    at plumbing.callback (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\web\chat-bot-telegram\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:183:22)
    at Request.emit (node:events:518:28)
    at Request.onRequestError (C:\web\chat-bot-telegram\node_modules\@cypress\request\request.js:869:8)
    at ClientRequest.emit (node:events:518:28)
    at _destroy (node:_http_client:880:13)
    at onSocketNT (node:_http_client:900:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
